src/utils/settingsUtil.ts

```diff
@@ -82,10 +84,10 @@ export const PreviewType = {
 export class SettingUtil {
 	/**
 	 * @description Get the current settings JSON.
-	 * @returns {ILivePreviewConfigItem} the LiveServerConfigItem, which is a JSON object with all of the settings for Live Preview.
+	 * @returns {ILivePreviewConfigItem} a JSON object with all of the settings for Live Preview.
 	 */
-	public static GetConfig(): ILivePreviewConfigItem {
-		const config = vscode.workspace.getConfiguration(SETTINGS_SECTION_ID);
+	public static GetConfig(scope?: vscode.ConfigurationScope): ILivePreviewConfigItem {

```

Comment:
first time introducing a `scope:resource` setting for the extension, so need to specify scope when getting config.